app/code/Magento/Pwa/Test/Mftf/ActionGroup/VeniaCartPageActionGroup.xml

```diff
@@ -68,13 +69,13 @@
         <fillField  selector="{{CartPageCouponCodeSection.couponCode}}" userInput="{{discount.CouponCode}}" stepKey="enterCouponCode"/>
         <wait time="1" stepKey="wait4"/>
         <click selector="{{CartPageCouponCodeSection.applyCoupon}}" stepKey="applyCouponCode"/>
-        <wait time="5" stepKey="wait5"/>
+        <wait time="10" stepKey="wait5"/>
```

Comment:
You can find more info here https://devdocs.magento.com/mftf/docs/test/actions.html?itm_source=devdocs&itm_medium=quick_search&itm_campaign=federated_search&itm_term=stepke 

stepKeys are basically info about the action. In case of `wait` action it's already known that script is waiting. wait..2..3...4 are just to make it unique, order does not really matter.